platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/configurationcache/problems/ConfigurationCacheProblems.kt

```diff
@@ -138,14 +138,35 @@ class ConfigurationCacheProblems(
         return "configuration-cache"
     }
 
+    fun reportFailure(summary: Summary = summarizer.get(), htmlReportFile: File? = null): Throwable? {
```

Comment:
💅 I'd use a more passive name here since this function by itself doesn't have a side effect (`report` makes me think it has), `queryFailure`?

Reply:
`queryFailure` it is.